### DISCUSSION TITLE
nuget-package-restore-with-multiple-repositories-and-file-shares/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#47] [Blog comments]
# nuget-package-restore-with-multiple-repositories-and-file-shares/

https://blogs.taiga.nl/martijn/posts/nuget-package-restore-with-multiple-repositories-and-file-shares/

**Comment by disqus-comments-migrator** [comment]
<em>Alexander M. Batishchev commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at juli 20 2013, 06:36</em>

---

<p>Thanks for pointing out on slashes espace!</p>
